<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="hash_disp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display hash table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_disp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Hash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hash_disp1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display hash table1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_disp_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display file 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Hash 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>351</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hash_disp2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display hash table2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_disp3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display file 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Hash 2</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hash_disp3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display hash table3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_disp4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display file 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Hash3</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>1001</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>search</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
